Utilities/AzureOnboardingCLI/azure-automatic-role-assignment.py

```diff
@@ -367,41 +461,62 @@ def main():
     )
     args = parser.parse_args()
 
-    subscription = args.subscription
+    subscription_id = args.subscription
+    if subscription_id is None:
+        print("INFO: No --subscription specified, defaulting to tenant scope.")
+
+    products = args.products
+    if products is None:
+        print("INFO: Defaulting to all products.")
+        products = f"{Products.CORE} {Products.COST_INTELLIGENCE}"
```

Comment:
In the spirit of "least surprise", we should default to CORE only, right?

Reply:
I'm ok with that.